LICENSE.md

```diff
@@ -1,2 +1,4 @@
-Code in this repository is licensed under [Apache License Version 2.0](LICENSE-code) (SPDX-License-Identifier: Apache-2.0).
-Documentation in this repository is licensed under [Creative Common Attribution 4.0 International License](LICENSE-document) (SPDX-License-Identifier: CC-BY-4.0)
\ No newline at end of file
+Code in this repository is licensed under [Apache License Version
+2.0](LICENSE-code) (SPDX-License-Identifier: Apache-2.0). Documentation in this
+repository is licensed under [Creative Common Attribution 4.0 International
+License](LICENSE-document) (SPDX-License-Identifier: CC-BY-4.0)
```

Comment:
Hmm i think we may have to special case this file? In case it isn't parsed correctly by scanners/github? Maybe we could have an exception for this file.